packages/language-server/server/constants.js

```diff
@@ -0,0 +1,6 @@
+const EMPTY_SCOPE = {
```

Comment:
May we make `EMPTY_SCOPE` as an instance of `TypeScope` for `EMPTY_TYPE_SCOPE`, and `VariableScope` for `EMPTY_VARIABLE_SCOPE`?

Reply:
I made it. Thank you for your review)